src/components/MultiHopTrade/components/LimitOrder/hooks/useLimitOrders.tsx

```diff
@@ -30,7 +29,7 @@ export const useLimitOrders = () => {
   const toast = useToast()
   const translate = useTranslate()
   const prevLimitOrdersData = usePrevious(limitOrdersQuery.currentData)
-  const actions = useAppSelector(actionSlice.selectors.selectActions)
+  const actions = useAppSelector(selectActions)
```

Comment:
Hell yeah memoization 🐐 

Wondering, to make that even better, do we want to add `selectLimitOrderActions` as a selector on top of `selectActions`